documentation/API.md

```diff
@@ -396,6 +428,44 @@ returns:
 }
 ```
 
+#### Manage localisations
+
+Used by the front-end `/admin/localisations` page
+
+POST: `/enable-language?code=<languageCode>&enabled=<true/false>`
+- To enable or disable a language that is already installed. If parameter `enabled` is omitted, the current setting will be toggled.
+
+POST: `/install-language`
+- To install a a new language into the system.
+
+**Input parameters** (as body JSON) example:
+```
+{
+    "language": {
+        "languageName": "Portugese",
+        "description": "Portugese translation for Angola users",
+        "code": "pt_an",
+        "flag": "🇦🇴",
```

Comment:
Two points here: 
1 - Not sure if good to add this information here. It might be sensitive? Might be worth to use a less common example... Perhaps Tetum - so we keep the same approach as in mSupply mobile?
2 - Portugese -> Portuguese

Reply:
Yeah, good point. I changed the example to just a generic "Portuguese" with Brazil flag. No mention of Angola now.

3bca97b1

> 2 - Portugese -> Portuguese

😲

I've been doing it wrong my whole life 🤦‍♂️

Reply:
😆 Oh that's funny!